config/schema/web_page_archive.schema.yml

```diff
@@ -21,14 +21,37 @@ web_page_archive.web_page_archive.*:
       sequence:
         type: mapping
         mapping:
-          uuid:
-            type: string
           id:
             type: string
+          data:
+            type: web_page_archive.capture_utility.[%parent.id]
+          weight:
+            type: integer
+          uuid:
+            type: string
     run_entity:
       type: string
       label: 'Run Entity'
 
+web_page_archive.capture_utility.*:
+  type: mapping
+  label: 'Capture utility settings'
+
+web_page_archive.capture_utility.html_capture_utility:
+  type: mapping
+  label: 'HTML Capture utility'
+  mapping:
+    capture:
+      type: boolean
+      label: 'Capture?'
```

Comment:
It maps to the checkbox added by `HTMLCaptureUtility:: buildConfigurationForm`

I am cool to drop it, but this is how I directly linked the form element to the schema field.